goplay/src/main/java/be/tapped/goplay/content/models.kt

```diff
@@ -0,0 +1,94 @@
+package be.tapped.goplay.content
+
+import kotlinx.serialization.Serializable
+
+// TODO discuss the duplication here
```

Comment:
The model for the overview and the details is effectively the same.  
However the model for the  overview (`goplay.be/programmas` ), only returns a subset of the fields.

Reply:
Ideally I don't want to have the repetition, but we might want to put that responsibility in the layer above.

Reply:
Actually I made it part of a sealed interface hierarchy to avoid the duplication and KotlinX seems to be fine with this 😀